src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -416,11 +417,13 @@ private GridPane setUpEditWorkGridPane(final Work work, final Dialog<Work> dialo
       return grid;
    }
 
-   private Button createCopyProjectButton(final List<Work> projectWork) {
-      final Button copyButton = new Button("", SvgNodeProvider.getSvgNodeWithScale(RESOURCE.SVG_CLIPBOARD, 0.03, 0.03));
-      copyButton.setMaxSize(20, 18);
-      copyButton.setMinSize(20, 18);
-      copyButton.setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
+   private Button createCopyProjectNotesButton(final List<Work> projectWork) {
+      final Button copyNotesButton = new Button("", SvgNodeProvider.getSvgNodeWithScale(RESOURCE.SVG_CLIPBOARD, 0.03, 0.03));
```

Comment:
When using it the new icon feels more like "copy all child items" for me, as it has multiple instances

![image](https://github.com/doubleSlashde/KeepTime/assets/5772511/4c4ea8db-fc4b-4e8a-b45a-4635ed3df9fa)

what do you think? I think I would exchange the icons. So the "clipboard" copies the current row and the "multiple clipboards" copies the children.